iron-dropdown-scroll-manager.html

```diff
@@ -41,6 +41,10 @@
        */
       elementIsScrollLocked: function(element) {
         var currentLockingElement = this.currentLockingElement;
+
+        if (currentLockingElement === undefined)
```

Comment:
Neither. If you have `allowOutsideScroll = true`, then you don't push anything on the scroll stack (https://github.com/PolymerElements/iron-dropdown/blob/master/iron-dropdown.html#L297), so `lockingElements` is in a bad state (https://github.com/PolymerElements/iron-dropdown/blob/master/iron-dropdown-scroll-manager.html#L82)